SpecFlowProject10/StepDefinitions/CalculatorStepDefinitions.cs

```diff
@@ -9,83 +9,82 @@
 using NUnit.Framework;
 using UnitTestProject1.Elements;
 using UnitTestProject1.appl;
-using UnitTestProject1.CalculatorVar;
 using TestStack.White.UIItems.WPFUIItems;
 using Test.Form;
 using TechTalk.SpecFlow.CommonModels;
+using Test.Configuration;
+
 
 namespace SpecFlowProject10.StepDefinitions
 {
     [Binding]
     public class CalculatorStepDefinitions
     {
-        Customvar calculatorvar1 = new Customvar();
         CalculatorForm calculator = new CalculatorForm(SearchCriteria.All, "");
+        string smode;
 
-        [Given(@": Open  Calculator Application and")]
-        public void GivenOpenCalculatorApplicationAnd()
+        [Given(@"Open the Calculator Application")]
+        public void GivenOpenTheCalculatorApplication()
         {
-
-            string path = calculatorvar1.calpath();// All variable class
-            AppManager.LanuchApp("Calculator");
-
+            string path = Configuration.path("calculator");
+            AppManager.LanuchApp(path);
         }
 
-        [Given(@": Click on View Menu and select the Standard Option")]
-        public void GivenClickOnViewMenuAndSelectTheStandardOption()
+        [Given(@": Select the Mode '([^']*)'")]
```

Comment:
Remove ':' from step definitions
Regular can be simplified '([^']*)' -> '(.\*)'
Check all steps

Reply:
changed